cmake/pcl_options.cmake

```diff
@@ -66,7 +66,7 @@ option(BUILD_tools "Useful PCL-based command line tools" ON)
 option(WITH_DOCS "Build doxygen documentation" OFF)
 
 # set index size
-set(PCL_INDEX_SIZE -1 CACHE STRING "Set index size. Available options are: 8 16 32 64")
+set(PCL_INDEX_SIZE -1 CACHE STRING "Set index size. Available options are: 8 16 32 64. A negative value indicates default size (32 for PCL >= 1.12, sizeof(int) otherwise)")
```

Comment:
```suggestion
set(PCL_INDEX_SIZE -1 CACHE STRING "Set index size. Available options are: 8 16 32 64. A negative value indicates default size (32 for PCL >= 1.12, 8*sizeof(int) i.e., the number of bits in int, otherwise)")
```